pkgs/development/tools/guestfs-tools/default.nix

```diff
@@ -0,0 +1,116 @@
+{ lib
+, stdenv
+, fetchurl
+, autoreconfHook
+, bison
+, cdrkit
+, cpio
+, flex
+, getopt
+, makeWrapper
+, pkg-config
+, qemu
+, ocamlPackages
+, perlPackages
+, bash-completion
+, jansson
+, libvirt
+, libxcrypt
+, libxml2
+, ncurses
+, pcre2
+, xz
+, hivex
+, libguestfs
+, guestfs-tools
+, callPackage
+, appliance ? null
```

Comment:
That's a direct adaption of how things are done in libguestfs. Maybe there are better solutions? I'm especially not fond of the explicit overrides for the tests.